### PR TITLE
Updating documentation for Placement and JobOrder

### PR DESCRIPTION
**Pull request by tiffanineyland**
- Adding jobLocation, location, and owner to Placement entity
- Updating description for owner on Placement and Job to note that the field map name will be userID/jobUserID

### CODE REVIEW

**Review by JorgeBerroa — APPROVED** [review]

File: source/includes/entityref/_placement.md [hunk 1]
```diff
@@ -61,9 +61,11 @@ Represents a successfully filled job; for example, a placement of a candidate in
...
 | invoiceID | Integer | **Not supported in this release.** Id of the Invoice object associated with this Placement. This field is used for Permanent placements since only a single invoice is needed. | | |
 | invoiceItems | To-many association | **Not supported in this release.** Invoice Items associated with this Placement. | | |
+| jobLocation | To-one association | Location sourced from the associated JobOrder record. | | |
 | jobOrder | To-one association | JobOrder associated with this Placement. Cannot be changed after this record is created. | X | |
 | jobSubmission | To-one association | JobSubmission associated with this Placement. | | |
 | lastBteSyncDate | Timestamp | Pay and Bill corporations only: Used to store the date and time of the last known sync a Bullhorn placement was synced into Bullhorn Time & Expense. This field should only appear if a corporation is WFR enabled and PeopleNet Front Office Integration enabled. | | X |
+| location | To-one association | Location sourced from the associated ClientCorporation record associated to this placement by way of the Location entity. | | |
```
Comment (Allblues) [inline source/includes/entityref/_placement.md:68]: First letter capital on Placement

File: source/includes/entityref/_placement.md [hunk 2]
```diff
@@ -77,6 +79,7 @@ Represents a successfully filled job; for example, a placement of a candidate in
 | otherHourlyFeeComments | String (2147483647) | Free text field for comments on additional hourly fees. | | |
 | overtimeMarkupPercentage | Double | overtimeMarkupPercentage = (clientOvertimeRate - overtimeRate) / overtimeRate. Multiply by 100 to get actual percentage. | | |
 | overtimeRate | Double | Hourly rate at which the employee will be paid for overtime work. | | |
+| owner | To-one association | CorporateUser who owns the JobOrder. The default value is the user who creates the JobOrder and is also the default owner of Placement records created from this job. The field name in ATS for owner is listed in field maps as jobUserID. | | |
```
Comment (Allblues) [inline source/includes/entityref/_placement.md:82]: Change "job" to "JobOrder"

**Review by Allblues — CHANGES_REQUESTED** [review]

File: source/includes/entityref/_placement.md (hunk 1 above)
Comment (tiffanineyland) [reply source/includes/entityref/_placement.md]: Capitalized first letter in Placement

File: source/includes/entityref/_placement.md (hunk 2 above)
Comment (tiffanineyland) [reply source/includes/entityref/_placement.md]: Changed job to JobOrder

**Review by Allblues — APPROVED** [review]

**Pull request merged**